app/mailers/request_mailer.rb

```diff
@@ -0,0 +1,24 @@
+class RequestMailer < ApplicationMailer
+  def request_cancel_partner_notification(request_id:)
+    @request ||= Request.find(request_id)
+    @organization ||= @request.organization
+    @partner = @request.partner
+
+    # Generate the request_items in a formatted way so
+    # I can render Item names
+    item_ids = @request.request_items.map { |r| r['item_id'] }
+    items = Item.where(id: item_ids)
```

Comment:
I would add `.index_by(&:id)` here and then key into that hash on line 13 instead of using a `find`

Reply:
Great idea! I like it